### DISCUSSION TITLE
Changing rpl-udp contiki-ng Makefile

### DISCUSSION THREAD

**Discussion by Jawaheralharbi** [whitefield-framework/whitefield#122] [Q&A] [answered]
I have changed the makefile setup in rpl-udp, but when I start the Whitefield, nothing changed. How can I run the simulation against the new makefile changes? (I have executed the make command in the makefile folder and it was successful).

<img width="387" alt="Screenshot 2021-05-31 at 9 17 40 am" src="https://user-images.githubusercontent.com/50404124/120199215-e9a86e80-c21a-11eb-9ccf-e08492311d8a.png">
<img width="602" alt="Screenshot 2021-05-31 at 9 13 04 am" src="https://user-images.githubusercontent.com/50404124/120199241-ee6d2280-c21a-11eb-8e45-b290b6b9b0ff.png">

**Comment by nyrahul** [comment, marked as answer]
There are two options `make contiking_clean && make`. This will clean up contiking and recompile everything.

Or a total clean up option is to do: `make allclean && make`
